opm/simulators/flow/GenericTracerModel_impl.hpp

```diff
@@ -445,16 +445,16 @@ linearSolveBatchwise_(const TracerMatrix& M, std::vector<TracerVector>& x, std::
     int maxIter = 100;
 
     int verbosity = 0;
-    PropertyTree prm;
-    prm.put("maxiter", maxIter);
-    prm.put("tol", tolerance);
-    prm.put("verbosity", verbosity);
-    prm.put("solver", std::string("bicgstab"));
-    prm.put("preconditioner.type", std::string("ParOverILU0"));
 
 #if HAVE_MPI
-    if(gridView_.grid().comm().size() > 1)
+    if (gridView_.grid().comm().size() > 1)
```

Comment:
Shouldn't the if-branch do the correct thing even without MPI? The overhead introduced should be negligble.